docs/src/examples/components/Form/Usage/FormExampleDropdown.shorthand.tsx

```diff
@@ -7,12 +7,17 @@ const fields = [
     label: { content: `Your best friend's name is:`, id: labelId },
     name: 'chooseFriend',
     key: 'choose-friend',
+    id: 'choose-friend-shorthand',
     control: {
       as: Dropdown,
       items: ['John Doe', 'Dohn Joe', 'John Joe', 'Dohn Doe'],
       'aria-labelledby': labelId,
       search: true,
       placeholder: 'Choose a friend',
+      searchInput: {
+        id: 'choose-friend-shorthand', // id needs to end up on the search input.
+      },
+      id: undefined, // not on the main wrapper element.
```

Comment:
This thing looks like very strange. Specify `id`s in three places to make it working?

Reply:
I don't see another choice, the most we can do is explain why.

That ID needs to end up on the `searchInput` in this case, since that is what we want to be focused on label click.

The `undefined` part is a bit sad since our `Form` assumes that the `id` needs to end up on the wrapper.

Any other ideas?

Reply:
I think the whole integration of our "input" components with Form does not work as expected - Dropdown searchable/non-searchable, checkbox, radio - all work differently and need some workarounds. @silviuavram can you spend some time thinking about a better proposal? @bmdalex (hi) had a proposal to use

```
<Form.Field control={Dropdown} />
```

That way, Stardust can at least understand what is the underlying input (`if control===Dropdown`) and pass the id in a correct way. It still won't work for wrapped or custom components, but at least it would work out of the box for the most common use cases. What do you think @layershifter ?

Reply:
I will look into this. Maybe our input components should provide a label API. Can we close this ticket?

Reply:
It's how it's done in SUIR.

We discussed this on our of our syncs and agreed that it will a better option.